utilities/test_suite/readme.md

```diff
@@ -0,0 +1,19 @@
+RPP Test Suite
```

Comment:
AMD Radeon Performance Primitives (RPP) Test Suite

The RPP Test Suite is a set of tests that are used to validate the functionality and performance of the AMD Radeon Performance Primitives vision library.

Features:
The suite includes:
- Unit tests that execute the desired functionality and variant once, report RPP execution wall timee and save output images.
- Performance tests that execute the desired functionality and variant 100 times by default, and report max/min/avg RPP execution wall time, or optionally, AMD rocprof kernel profiler max/min/avg time for HIP backend variants.
- Unit and Performance tests are included for three layouts - PLN1 (1 channel planar NCHW), PLN3 (3 channel planar NCHW) and PKD3 (3 channel packed/interrleaved NHWC).
- Unit and Performance tests are included for various input/output bitdepths including U8/F32/F16/I8.
- Support for pixelwise output referencing against golden outputs, and functionality validation checking, by tolerance-based pass/fail criterions for each variant. (Current support for U8 PKD3 variants)

Reply:
Updated readme